x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -113,6 +117,45 @@ export function generateMappings(fields: Field[]): Mappings {
   return { properties: props };
 }
 
+function generateMultiFields(fields: Fields): MultiFields {
+  const multiFields: MultiFields = {};
+  if (fields) {
+    fields.forEach((f: Field) => {
+      const type = f.type;
+      switch (type) {
```

Comment:
What happens if we don't know the type? Should we log an error or similar so at least the operators knows something was not as expected and we can debug it? I remember we had a similar discussion in the past with an other feature.

Reply:
Good point. This is https://github.com/elastic/kibana/issues/63837 now.